src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -272,6 +251,7 @@ const useCoreFeed = () => {
     onDeleteBlockedPostCompleted,
     saveSearchHeight,
     sendViewPostTime,
+    sendViewPostTimeWithFeeds,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Exposing `sendViewPostTimeWithFeeds` in the return statement is a good practice as it allows other components or hooks to use this function if needed.
```diff
+ sendViewPostTimeWithFeeds,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed